tests/test_search.py

```diff
@@ -455,3 +456,10 @@ def test_cbers_awfi_valid(list_directory, session):
         'version': '4'}]
 
     assert list(search.cbers(path, row, sensor)) == expected
+
+
+def test_s2_date_filter():
+    start_date = datetime(2017, 1, 1)
+    end_date = datetime(2017, 5, 15)
+    results_date_filter = list(search.sentinel2(22, "K", "HV", start_date=start_date, end_date=end_date))
+    assert len(results_date_filter) == 22
```

Comment:
@pablotcarreira Can you add some tests for https://github.com/RemotePixel/aws-sat-api-py/pull/6/files#diff-fc9a6edf050c18c92ddbbaa9b1ccefc0R179 and https://github.com/RemotePixel/aws-sat-api-py/pull/6/files#diff-fc9a6edf050c18c92ddbbaa9b1ccefc0R182